Assets/GothicVR/Scripts/Data/SkyLayerData.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+
+namespace GVR.Data
+{
+    public class SkyLayerData
+    {
+        public Texture2D[] texBox = null;
```

Comment:
Please don't set default values if they are the default ones from C# anyways. (Rider will mark them in grey as not needed anyways).